Rename workflow and add WP-CLI ingestion to the workflow

dbeja opened the pull request:
This PR adds the `.yml` extension to the WP-Cli workflow so it becomes active and implements the ingestion of the WP-Cli commands fetched into the database.

This way, every day at 6am after fetching the full list of WP-CLI commands, they are automatically sent to the database, so the database has the latest version of them.

Closes #31.

dbeja committed:
Rename update-wp-cli-commands to update-wp-cli-commands.yml

dbeja committed:
Ingest WP-CLI commands list

heiglandreas reviewed (commented):
Do we really need to do the commit with the result of the cmd-dump?

Can't we improve that by adding it to the worker-cache or - perhaps easier - by adding the `wp cli cmd-dump > ${{ github.workspace }}/cli/data/wpcli-commands.json` to the `cd cli && composer install && ./bin/docsdangit ingest -s wp-cli` so that the whole commandline would read something like this:

```bash
 wp cli cmd-dump > ${{ github.workspace }}/cli/data/wpcli-commands.json && cd cli && composer install && ./bin/docsdangit ingest -s wp-cli`
```

Or am I thinking too easy?

dbeja committed:
Add wp-cli dump and version path args to cli

dbeja committed:
Remove commit and add paths args to cli

dbeja commented:
@heiglandreas I agree with you, there's no need for the commit now that we do everything from the GitHub Action.

I've removed the commit and also added 2 new params to the cli tool so we can define where to parse the dump file and the wp cli version and this way I'm using a temporary directory of the worker for that.
